jobs/parser/spec

```diff
@@ -33,6 +33,8 @@ properties:
   logstash_parser.filters:
     description: "The configuration to embed into the logstash filters section"
     default: ''
+  logstash_parser.inputs:
+    description: "The configuration to embed into the logstash inputs section"
```

Comment:
@axelaris, @dpb587, Last thing to do before we merge this - write some documentation.

Any idea how we can format the documentation here is the spec file so it looks good at http://bosh.io/jobs/parser?source=github.com/logsearch/logsearch-boshrelease&version=22#p=logstash_parser.outputs ?